template/ios/RNBaseProjectTypeScript.xcodeproj/xcshareddata/xcschemes/RNBaseProjectTypeScript.xcscheme

```diff
@@ -41,7 +50,7 @@
       </Testables>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Debug"
+      buildConfiguration = "Release"
```

Comment:
I think we should run debug on the first run of the app

Reply:
RNBaseProjectTypeScript.xcscheme is scheme use for build production so the buildConfiguration is Release. If you want to run on debug, you can choose RNBaseProjectTypeScriptDev.xcscheme